COMPILING.md

```diff
@@ -1,19 +1,125 @@
 # WHAT ARCHITECTURE?
 
-CPROVER now needs a C++11 compliant compiler and works in the following
-environments:
+CPROVER now needs a C++11 compliant compiler and is known to work in the
+following environments:
 
 - Linux
 - MacOS X
+- Microsoft Visual Studio 2019
+
+The above environments are currently tested as part of our continuous
+integration system. It separately tests both the CMake build system and the
+hand-written make files. The latest build steps being used in CI can be
+[referenced here](https://github.com/diffblue/cbmc/blob/develop/.github/workflows/pull-request-checks.yaml).
+
+The environments below have been used successfully in the
+past, but are not actively tested:
+
 - Solaris 11
 - FreeBSD 11
 - Cygwin
-- Microsoft Visual Studio
 
-The rest of this document is split up into three parts: compilation on Linux,
-MacOS, Windows.  Please read the section appropriate for your machine.
+# Building using CMAKE
```

Comment:
Would it be possible we have `CMake` consistently capitalised?

Reply:
Done.